### ISSUE TITLE
Accessibility: active category is not announced to screen readers

### ISSUE THREAD

**Issue by philippsauer** [intershop/intershop-pwa#1832]
**Actual Behavior**

When navigating through the filter bar (CategoryNavigationComponent) the active category is visually highlighted (see screenshot), but it is not announced if the item is active.

**Expected Behavior**

The active category should be announced when tabbing through the category navigation tree.

Suggestion: add the following line to the a-tag of the filter item:
`[attr.aria-current]="category.uniqueId === currentCategoryId ? 'true' : undefined"`

**Steps to Reproduce the Bug**

1. Go to https://intershoppwa-b2b.azurewebsites.net/computers/notebooks-and-pcs-updated/notebooks-ctgComputers.1835.151
2. Tab through the category navigation in the sidebar.

**Additional Context, like Screenshots, Log File Snippets etc.**

Tested using NVDA. 

![Image](https://github.com/user-attachments/assets/29716755-b242-4a4f-8c95-b78d74c0ae39)

[AB#107449](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/107449)
